src/controllers/krate/versions.rs

```diff
@@ -302,4 +355,208 @@ struct PaginatedVersionsAndPublishers {
 struct ResponseMeta {
     total: i64,
     next_page: Option<String>,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    release_tracks: Option<ReleaseTracks>,
+}
+
+#[derive(Debug, Eq, PartialEq, Serialize)]
+struct ReleaseTracks(IndexMap<ReleaseSlug, HighestSemver>);
```

Comment:
```suggestion
struct ReleaseTracks(IndexMap<ReleaseTrackName, ReleaseTrackDetails>);
```

the other nitpick: what do you think about names like this to make it clearer that these are all related to the release tracks field?

Reply:
This seems acceptable to me, although the name detail is not very self-explanatory. However, the content is straightforward, so I think it should be fine.

Reply:
> although the name detail is not very self-explanatory

that is true, but `HighestSemver` assumes that we will never add any other fields to the struct and I'm not sure whether that will stay true forever :D

Reply:
yeah, fair enough!